billing-ui/src/pages/Report/Report.js

```diff
@@ -123,12 +123,14 @@ const Report = ({ history }) => {
     const [entriesToDisplay, setEntriesToDisplay] = useState([]);
     const [reportSummary, setReportSummary] = useState([]);
     const [projects, setProjects] = useState(noProjects(report));
+    const [CSVFileName, setCSVFileName] = useState('');
     const [filters, setFilters] = useState({
         bucketSize: defaultBucketSize,
         fromDate: moment().subtract(1, 'months').startOf('day'),
         projects: [],
         toDate: moment().startOf('day'),
     });
+    const [canGenerateReport, setCanGenerateReport] = useState(filters.fromDate && filters.toDate);
```

Comment:
Let's disable the "generate report" button, if any of the necessary filters is missing.